src/main/java/xbot/common/subsystems/drive/swerve/SwerveSteeringSubsystem.java

```diff
@@ -352,14 +336,14 @@ public void setMotorControllerPidTarget() {
             Angle currentPosition = getBestEncoderPosition();
             Angle angleBetweenDesiredAndCurrent = Degrees.of(MathUtil.inputModulus(target.minus(currentPosition).in(Degrees), -90, 90));
             aKitLog.record("angleBetweenDesiredAndCurrent-Degrees", angleBetweenDesiredAndCurrent.in(Degrees));
-            aKitLog.record("MotorControllerPosition-Rotations", this.motorController.getPosition().in(Rotations));
+            aKitLog.record("MotorControllerPosition-Rotations", getMotorController().get().getPosition().in(Rotations));
 
-            Angle targetPosition = this.motorController.getPosition().plus(
+            Angle targetPosition = getMotorController().get().getPosition().plus(
                     Rotations.of(angleBetweenDesiredAndCurrent.in(Degrees) / degreesPerMotorRotation.get())
             );
 
             aKitLog.record("TargetPosition-Rotations", targetPosition.in(Rotations));
-            this.motorController.setPositionTarget(targetPosition, XCANMotorController.MotorPidMode.Voltage, 0);
+            getMotorController().get().setPositionTarget(targetPosition, XCANMotorController.MotorPidMode.Voltage, 0);
```

Comment:
⭐  I do like the advice I saw that `get()` should basically never be called and it's better to call the more explicitly scary `orElseThrow()`

Reply:
and/or alternatively, at the top of this block maybe cache the motorController like `var motor = this.motorController.orElseThrow()` and then use that motor value in all these places